include/AAInterface.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "openauto/Service/IAndroidAutoInterface.hpp"
+#include "app/action.hpp"
+
+
+
+class AAInterface : public QObject, public openauto::service::IAndroidAutoInterface {
+    Q_OBJECT
+
+   public:
+    AAInterface();
+    ~AAInterface();
+    void mediaPlaybackUpdate(const aasdk::proto::messages::MediaInfoChannelPlaybackData& playback) override;
```

Comment:
for consistency lets keep these snake case

Reply:
Can't do this without another openauto PR - these are overrides of the Interface (which was styled after openauto's code style)

Reply:
yep missed that keep it like it is then